product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -48,6 +48,64 @@ Note: In this case, if the grace period is kept too high then writes will contin
 
 Having said that, most of the time, upon loss of the current Raft leader the new Raft leader gets elected by BDR within a few seconds provided more than half of the nodes (quorum) are still up. Hence, if the Raft leader is down but the write leader is still up then proxy can be configured to allow routing by keeping `consensus_grace_period` to a non-zero, positive value. The proxy will wait for the Raft leader to get elected during this period before stopping routing. This may be helpful in some cases where availability is more important.
 
+### Proxy health check
+
+PGD Proxy provides the following HTTP(s) health check API endpoints. These APIs are `GET` requests and do not require a request body.
+```
+GET /health/is-live
+GET /health/is-ready
+```
+
+#### Liveness
+
+Liveness check either return `true` with HTTP status code `200 (OK)` or an error but never `false`. This is due to the fact that Proxy service exits immediately effectively stopping HTTP server if it finds a critical error on startup that cannot be recovered or prevents further operations.
+
+#### Readiness
+
+On receiving this request, proxy checks if it can successfully route traffic through the `proxy.endpoint`. If the check returns successfully then API returns `true` with HTTP status code `200 (OK)` else `false` with HTTP status code `500 (Internal Server Error)`. It will return `false` in the following scenarios.
+* Proxy detects that the Raft leader is not available and `consensus_grace_period` has expired.
+* Proxy detects that the write leader is not available.
+* Proxy cannot establish a TCP connection to the current write leader
+* The TLS certificate on the current write leader is expired
+
+It is important to know that once started proxy always remains in available state even if this check return false because the proxy can recover itself automatically (self heal) once the BDR cluster or connectivity is restored.
+
+!!! Note
+    Proxy will perform the readiness check against the backend Postgres only when the readiness endpoint is requested and not on periodic basis.
+
+#### Configuration
```

Comment:
Move this section under [Configuration](https://www.enterprisedb.com/docs/pgd/latest/routing/proxy/#proxy-log-location), after [PGD proxy user](https://www.enterprisedb.com/docs/pgd/latest/routing/proxy/#proxy-log-location) section and before PGD Proxy service. Saying only required if health check probes are required in environment like k8s.

On this page, here, only add link to that configuration section saying see this for how to enable these endpoints etc. 

Similarly, same link to this page from Configuration section.

Reply:
done